### PR TITLE
[RUN-5625] Propagate events from BrowserView to Application and System

### PR DESCRIPTION
**Pull request by MichaelMCoates**
#### Description of Change

Sets up event propagation from BrowserView up to Application and System.

Tests:
https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5d88dfaf7ba87401eaf565df

#### Checklist

- [X] PR description included and stakeholders cc'd
- [X] `npm test` passes
- [X] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [X] Integration tests added in this repo and in the [test dashboard](https://testing-dashboard.openfin.co/#/app/dashboard)
- [ ] manual tests are [changed or added](https://github.com/openfin/test_apps)
- [X] relevant documentation is changed or added
- [X] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [X] PR release notes describe the change in a way relevant to app-developers
- [X] PR has assigned reviewers

#### Release Notes

Notes: 

- BrowserView events now propagate up to Application and System.

### CODE REVIEW

**Review by pbaize — DISMISSED** [review]

File: tutorials/Application.EventEmitter.md [hunk 1]
```diff
@@ -450,3 +455,69 @@ Generated when a child window starts loading.
     uuid: "454C7F31-A915-4EA2-83F2-CFA655453C52" // the UUID of the application
 }
 ```
+
+#### view-created
+Generated when a child BrowserView is created.
```
Comment (rdepena) [inline tutorials/Application.EventEmitter.md]: We can drop "child Browserview" from this doc.

File: tutorials/System.EventEmitter.md [hunk 2]
```diff
@@ -353,3 +358,69 @@ Generated when a window gains focus.
     uuid: "454C7F31-A915-4EA2-83F2-CFA655453C52" // the UUID of the application
 }
 ```
+
+#### view-created
+Generated when a child BrowserView is created.
```
Comment (rdepena) [inline tutorials/System.EventEmitter.md]: Same here, drop "child".

**Review by rdepena — CHANGES_REQUESTED** [review]

File: tutorials/Application.EventEmitter.md [hunk 3]
```diff
@@ -450,3 +455,69 @@ Generated when a child window starts loading.
...
+```js
+//This response has the following shape:
+{
+    name: "BrowserViewName" // the name of the BrowserView
+    target: {uuid: 'windowUuid', name: 'windowName'}, // the window this BrowserView was attached to
+    topic: "view",
+    type: "destroyed",
+    uuid: "454C7F31-A915-4EA2-83F2-CFA655453C52" // the uuid of the BrowserView
+}
+```
```
Comment (licui3936) [inline tutorials/Application.EventEmitter.md:457]: You already add reference for above events( @tutorial BrowserView.EventEmitter).  So no need to add them again in application.EventEmitter.md.

File: tutorials/System.EventEmitter.md [hunk 4]
```diff
@@ -353,3 +358,69 @@ Generated when a window gains focus.
...
+```js
+//This response has the following shape:
+{
+    name: "BrowserViewName" // the name of the BrowserView
+    target: {uuid: 'windowUuid', name: 'windowName'}, // the window this BrowserView was attached to
+    topic: "view",
+    type: "destroyed",
+    uuid: "454C7F31-A915-4EA2-83F2-CFA655453C52" // the uuid of the BrowserView
+}
+```
```
Comment (licui3936) [inline tutorials/System.EventEmitter.md:360]: same here

File: tutorials/Application.EventEmitter.md (hunk 1 above)
Comment (MichaelMCoates) [reply tutorials/Application.EventEmitter.md]: done!

File: tutorials/System.EventEmitter.md (hunk 2 above)
Comment (MichaelMCoates) [reply tutorials/System.EventEmitter.md]: done!

File: tutorials/System.EventEmitter.md (hunk 4 above)
Comment (MichaelMCoates) [reply tutorials/System.EventEmitter.md:360]: Done!

File: tutorials/Application.EventEmitter.md (hunk 3 above)
Comment (MichaelMCoates) [reply tutorials/Application.EventEmitter.md:457]: Done!

**Review by pbaize — APPROVED** [review]

**Review by licui3936 — APPROVED** [review]

**Comment by MichaelMCoates** [comment]
@rdepena Any more changes?

**Review by rdepena — APPROVED** [review]

**Comment by rdepena** [comment]
Nope

**Pull request merged**